src/Sarif.Driver/Sdk/AnalyzeOptionsBase.cs

```diff
@@ -51,6 +51,12 @@ public class AnalyzeOptionsBase : IAnalyzeOptions
         public bool ComputeTargetsHash { get; set; }
 
         [Option(
+            'e',
+            "environment",
+            HelpText = "Log machine environment details of run to output file. WARNING: This option records potentially sensitive information (such as all environment variable values) to any emitted log.")]
+        public bool LogEnvironment { get; set; }
+
```

Comment:
We log other sensitive things (working directory, machine name...). Why does "environment" in particular get its own flag? Why not `--log-sensitive-information` that covers everything? Or alternatively, separate flags (`--log-environment`, `--log-machine-name`, etc.)?